docs/ruleset_functions/get_scc_skylight_roof_ratios.md

```diff
@@ -0,0 +1,67 @@
+
+## get_scc_skylight_roof_ratios
+
+Description: This function would determine skylight and envelope roof ratios for a building segment for residential, non-residential and semi-heated surface categories.  
+
```

Comment:
@xingcx and @mkarpman as previously discussed, this should be "skylight and envelope roof ratios for a building  for residential, non-resi..." aand not 'building segment', correct?

Reply:
Corrected.